addon/components/uk-badge.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@ember/component';
+import layout from '../templates/components/uk-label';
+import WidthMixin from 'ember-uikit/mixins/width';
+
+export default Component.extend(WidthMixin, {
+  layout,
+
+  tagName: 'span',
+
+  classNames: ['uk-badge'],
+
+  colorTemplate: 'uk-label-$color$'
```

Comment:
Why `colorTemplate` without `ColorMixin`? Is `badge` even colorable?

Reply:
fixed, copy pasta ;)